website/docs/index.html.markdown

```diff
@@ -22,6 +22,10 @@ own security trade-offs and caveats that are covered in the sections that
 follow. Consider these carefully before using this provider within your
 Terraform configuration.
 
+-> Visit the [Inject secrets into Terraform using the Vault provider](https://learn.hashicorp.com/tutorials/terraform/secrets-vault&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) Learn tutorial to learn how to use
```

Comment:
This link is a 404 for me. https://learn.hashicorp.com/tutorials/terraform/secrets-vault does work though